src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -99,10 +99,13 @@ export class DocumentSelectionRequestProcessor {
 
   	if (f2fSessionInfo.authSessionState === AuthSessionState.F2F_SESSION_CREATED && !f2fSessionInfo.yotiSessionId) {
 
-  		try {
+			try {
   			yotiSessionId = await this.createSessionGenerateInstructions(personDetails, f2fSessionInfo, postOfficeSelection, selectedDocument, countryCode);
   			await this.postToGovNotify(f2fSessionInfo.sessionId, yotiSessionId, personDetails);
   			await this.f2fService.updateSessionWithYotiIdAndStatus(f2fSessionInfo.sessionId, yotiSessionId, AuthSessionState.F2F_YOTI_SESSION_CREATED);
+				const updatedTtl = absoluteTimeNow() + Number(this.environmentVariables.authSessionTtlInSecs());
```

Comment:
could other vars like `authSessionTtlInSecs` be set to return number type pls so this conversion isn't needed